java/ql/integration-tests/all-platforms/java/diagnostics/no-build-system/test.py

```diff
@@ -0,0 +1,8 @@
+import os
+from create_database_utils import *
+from diagnostics_test_utils import *
```

Comment:
The CI jobs are failing with `ModuleNotFoundError: No module named 'diagnostics_test_utils'`. Did you forget to push the util file?

Reply:
This depends on an internal PR